components/idp-mgt/org.wso2.carbon.idp.mgt/src/test/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAOTest.java

```diff
@@ -1818,6 +1850,26 @@ private void addTestIdps() throws IdentityProviderManagementException {
         idPManagementDAO.addIdP(idp3, SAMPLE_TENANT_ID2);
     }
 
+    private void addTestIdps(Connection connection) throws IdentityProviderManagementException {
```

Comment:
Shall we add method comments?

Reply:
Update PR sent: https://github.com/wso2/carbon-identity-framework/pull/5538